projects/admin/src/app/service/record-routing.service.ts

```diff
@@ -295,17 +305,13 @@ export class RecordRoutingService {
     };
   }
 
-  private routeMatcher(url, type) {
+  private routeMatcher(url: any, type: string) {
     if (url[0].path === 'records' && url[1].path === type) {
       return this.matchedUrl(url);
     }
     return null;
   }
 
-  private cant() {
-    return of(false);
-  }
-
```

Comment:
I'm agree than it was no more used again but should be useful to keep it for future ?

Reply:
all resources will have access control, I think it's no longer necessary.